mmdet/models/roi_heads/bbox_heads/bbox_head.py

```diff
@@ -277,33 +277,63 @@ def get_bboxes(self,
                    cfg=None):
         if isinstance(cls_score, list):
             cls_score = sum(cls_score) / float(len(cls_score))
-        scores = F.softmax(cls_score, dim=1) if cls_score is not None else None
+
```

Comment:
add unitest for batch inference， especially for empty rois with shape (bs,0,5).

add the docstr of this file，expecially for `rois`